AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1088,7 +1088,13 @@ - (void)__layout
   // with negative sizes after applying margins, which will cause
   // measureWithSizeRange: on subnodes to assert.
   if (!CGRectEqualToRect(bounds, CGRectZero)) {
+    // Handle placeholder layer creation in case the size of the node changed after the initial placeholder layer
+    // was created
```

Comment:
@maicki very helpful description, thanks!  Not sure the easiest way to summarize for the comment but I think it's still a bit unclear.  You might just link to this PR with "details on the call behavior are at: " or something.